tests/decodeinputavformat.cpp

```diff
@@ -38,11 +38,11 @@ bool DecodeInputAvFormat::initInput(const char* fileName)
         goto error;
     uint32_t i;
     for (i = 0; i < m_format->nb_streams; ++i) {
-        AVCodecContext* ctx = m_format->streams[i]->codec;
```

Comment:
ffmpeg's api will change rapidly. We cannot follow very version. 
Other possible solution is make ffmpeg as a submodule of this project. We only update the submodule when we need new features.